data-generator.tpl.ts

```diff
@@ -252,6 +257,21 @@ export const DataGenerators: { [key in string]: ValueCreator } = {
       );
     },
   },
+  [DataType.BookImageUrlBytes]: {
+    createValue: (seed, refTime) => {
```

Comment:
Can this be a method instead of duplicated?

Reply:
I tried that, but the whole thing produced some cryptic errors when used in Start. I can try again.

Reply:
Done